assets/app/scripts/controllers/overview.js

```diff
@@ -15,8 +15,9 @@ angular.module('openshiftConsole')
     $scope.deployments = {};
     $scope.deploymentConfigs = {};
     $scope.builds = {};
-    $scope.images = {};
+    $scope.imageStreams = {};
     $scope.imagesByDockerReference = {};
+    $scope.imageStreamImageRefByDockerReference = {};
```

Comment:
comment here? "imageStreamImageRefByDockerReference lets us determine if a particular container's docker image reference belongs to an imageStream"